test/tap/add-remote-git-get-resolved.js

```diff
@@ -54,16 +61,16 @@ test('add-remote-git#get-resolved SSH', function (t) {
 })
 
 test('add-remote-git#get-resolved HTTPS', function (t) {
-  var getResolved = require('../../lib/cache/add-remote-git.js').getResolved
-
   verify('https://github.com/foo/repo')
   verify('https://github.com/foo/repo#master')
   verify('git+https://github.com/foo/repo.git#master')
   verify('git+https://github.com/foo/repo#decadacefadabade')
+  // DEPRECATED
```

Comment:
`// DEPRECATED: git doesn't know what to do with https://github.com:foo/repo.git` – future us is going to be confused if we don't include at least a brief explanation, now that we've removed the deprecation warning.